timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -70,6 +70,13 @@ export class TimeGraphChart extends TimeGraphChartLayer {
     private _mouseWheelHandler: { (ev: WheelEvent): void; (event: Event): void; (event: Event): void; };
     private _contextMenuHandler: { (e: MouseEvent): void; (event: Event): void; };
 
+    // Keep track of the most recently clicked point.
+    // If clicked again during _dbClickTime duration (milliseconds) trigger double click
+    private _recentlyClickedGlobal: PIXI.Point | null = null;
+    private _dbClickTime: number = 500;
+    private _mouseClicks = 0;
+    private _dblClickTimer: DebouncedFunc<() => void>;
```

Comment:
Maybe these should be called _multiClickTime/_multiClickTimer now that it's more than just doubleClick?